store/cache.go

```diff
@@ -100,13 +102,14 @@ func (cm *CacheManager) GetPath(cacheKey string) string {
 }
 
 // Get gets the given cacheKey content and puts it into targetPath
-func (cm *CacheManager) Get(cacheKey, targetPath string) error {
-	if err := os.Link(cm.path(cacheKey), targetPath); err != nil {
-		return err
+func (cm *CacheManager) Get(cacheKey, targetPath string) bool {
+	if err := os.Link(cm.path(cacheKey), targetPath); err != nil && !errors.Is(err, os.ErrExist) {
+		return false
```

Comment:
Why return a bool instead  of an error?

Reply:
Why would the cache return an error if the cache hit/miss decision was made below the cache interface? At that point all the user needs to know is whether there was a cache hit or miss and a bool accomplishes that